modules/editor/inserter/style.scss

```diff
@@ -0,0 +1,136 @@
+.inserter__button {
+	display: inline-block;
+	position: relative;
+	background: none;
+	border: none;
+	padding: 0;
+	margin-top: 20px;
+
+	a {
```

Comment:
>Would be good to start using classes for these.

And if we do add one, move styles out of nesting. ( _Edit:_ Saw @mtias left a similar comment below, but I'll not remove this so as to reinforce similar reaction 😉  )